lib/modules/manager/bazel-module/bazelrc.spec.ts

```diff
@@ -0,0 +1,206 @@
+import { codeBlock } from 'common-tags';
+import { fs } from '../../../../test/util';
+import { BazelOption, CommandEntry, ImportEntry, parse, read } from './bazelrc';
+
+jest.mock('../../../util/fs');
+
+function mockReadLocalFile(files: Record<string, string | null>) {
+  fs.readLocalFile.mockImplementation((file): Promise<any> => {
+    let content: string | null = '';
+    if (file in files) {
+      content = files[file];
+    } else {
+      return Promise.reject(new Error(`Unexpected file: ${file}`));
+    }
+    return Promise.resolve(content);
+  });
+}
+
+describe('modules/manager/bazel-module/bazelrc', () => {
+  describe('BazelOption', () => {
+    it.each`
+      a                                       | expArgs
+      ${'--show_timestamps'}                  | ${[['show_timestamps', undefined]]}
+      ${'--host_jvm_args=-XX:-UseParallelGC'} | ${[['host_jvm_args', '-XX:-UseParallelGC']]}
+      ${'--host_jvm_args='}                   | ${[['host_jvm_args', '']]}
+      ${'--jobs 600'}                         | ${[['jobs', '600']]}
+    `('parse($a)', ({ a, expArgs }) => {
+      const exp = (expArgs as [string, string | undefined][]).map(
+        (args) => new BazelOption(...args)
+      );
+      const result = BazelOption.parse(a);
+      expect(result).toEqual(exp);
+    });
+  });
+
+  describe('CommandEntry', () => {
+    it('getOption', () => {
+      const opt0 = new BazelOption('show_timestamps');
+      const opt1 = new BazelOption('keep_going');
+      const opt2 = new BazelOption('jobs', '600');
+      const cmdEntry = new CommandEntry('build', [opt0, opt1, opt2]);
+      expect(cmdEntry.getOption('does_not_exist')).toBeUndefined();
+      expect(cmdEntry.getOption(opt0.name)).toEqual(opt0);
+      expect(cmdEntry.getOption(opt2.name)).toEqual(opt2);
+    });
+  });
+
+  it('parse', () => {
```

Comment:
Add an error case for `--option_name=`

Reply:
The string `--option_name=` is legal. It sets the value to empty string. I have [a test for that case](https://github.com/renovatebot/renovate/pull/22563/files#diff-7d49d4f32eb84ccae03cb42376b37972e228998d7f440c5bea183c8ba542f843R25).